sample/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -13,6 +13,7 @@
 		97C146FC1CF9000F007C117D /* Main.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 97C146FA1CF9000F007C117D /* Main.storyboard */; };
```

Comment:
maybe we can delete this file from version control

Reply:
This file is needed to build iOS projects and must be included, otherwise we won't be able to build our project.